src/components/CurrentListing.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+
+//need an endpoint for listings in database
+//endpoint needs to make new instances of a Card/Listing component
+
+const Listing = () => {
+  return (
+    <div className="flex min-h-screen w-full  rounded-lg border-amber-600 border-2	border-solid	">
+      <div className="w-6/12 rounded-lg border-white border-2	border-solid">
```

Comment:
This is making two divs of white and orange borders just to see what the image and text next to it will look like. This will need to be updated when we get real images to work with